src/skin/legacyskinparser.cpp

```diff
@@ -553,6 +554,8 @@ QList<QWidget*> LegacySkinParser::parseNode(const QDomElement& node) {
         result = wrapWidget(parseEffectName(node));
     } else if (nodeName == "EffectSelector") {
         result = wrapWidget(parseEffectSelector(node));
+    } else if (nodeName == "EffectParameterKnobComposed") {
+        result = wrapWidget(parseEffectParameterKnobComposed(node));
```

Comment:
I haven't looked at it in detail but KnobComposed does appear to be different from the old Knob widget used by Shade.